packages/runtime/runtime-definitions/package.json

```diff
@@ -111,6 +111,19 @@
 		"typescript": "~5.4.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
```

Comment:
Just double-checking my own reasoning - these "forward compat" breaks mean that if someone is expecting to take in the old thing, and then after a package bump it's the new thing, everything will be fine.  BUT if someone were _implementing_ the old thing and took the package bump, they would see an error.

This is fine because these interfaces are not expected to be implemented by 1P partners, but only read from.

Right?

Reply:
Should we mark the relevant interfaces as `@sealed` then? I notice that at least some of them are not.